docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/libraries/charts/Chart.js

```diff
@@ -178,9 +178,6 @@ Chart.prototype = function () {
       console.log("applying base theme");
       // theme settings for NCI
       var theme = {
-          lang: {
-              thousandsSep: ','
-          },
```

Comment:
It's not clear why you removed this. Is this just a default value that HighCharts sets out of the box to the same value?

Reply:
Duplicate declaration on line 248.